r2-netty/build.gradle

```diff
@@ -1,12 +1,11 @@
 dependencies {
-  compile project(':pegasus-common')
   compile project(':data')
+  compile project(':pegasus-common')
+  compile project(path: ':pegasus-shaded-netty', configuration: 'shadow')
```

Comment:
Relying on special configurations from other projects is ok for test sources but I believe this will cause problems for main sources. Rest.li is published as a Maven (.pom) publication which doesn't support special configurations. For any consumers of `r2-netty`, the transitive dependency on `pegasus-shaded-netty` will likely be broken.

Reply:
Interesting, if true, we may need an alternative solution, @tjni Is there a way to test this part?

Reply:
I updated `publications.gradle` to upload the shaded artifact from li-netty. The POM that I inspected looks reasonable to me, but I'm not an expert. If either of you have suggestions for good steps to test that the artifact can be resolved, please let me know. Please feel free also to pull these changes down locally and inspect if you have some experience here.

Reply:
I also checked locally and the release looks good.